website/docs/design.md

```diff
@@ -76,7 +76,22 @@ type PackageManager interface {
 
 ## Tradeoffs
 
-- The core architectural choice of relying on packages as the unit of patching creates a couple of constraints:
-  - By relying on existing vulnerability scanner behavior that only detects vulnerabilities via presence/absence of vulnerable packages, copa is limited in the kinds of vulnerabilities it can address and false positive/negatives from scanners flow downstream to copa.
-  - copa depends on individual package manager adapters to correctly deploy patches to the target images, but there is a long tail of compatibility issues that arise depending on the target image itself (e.g. outdated package manager config/keys, invalid/missing package graph, etc.). Overall, the maintenance cost of the project is expected to be non-trivial to address this.
-- No support for windows containers given the dependency on buildkit.
\ No newline at end of file
+The core architectural choice of using packages as the unit of patching creates several constraints:
```

Comment:
we can remove this sentence since it is repetitive and the third tradeoff wouldn't fall under this

Reply:
@pradhans0906 thank you for the changes! just had this additional comment then LGTM